src/tools/rest-backend/config.cpp

```diff
@@ -0,0 +1,152 @@
+/**
+ * @file
+ *
+ * @brief configuration helper functions
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#include <iostream>
+#include <string>
+
+#include <config.hpp>
+
+namespace kdbrest
+{
+
+/* set dummy variables for statics */
+
+// JWT settings
+std::string Config::jwt_encryption_key = std::string ();
```

Comment:
Well that's obviously correct, but what is the advantage of making them member variables of a singleton? I only see an disadvantage and that is a more complicated call: `Config::instance().jwt_encryption_key` instead of `Config::jwt_encryption_key`...